src/main/java/com/google/devtools/build/lib/runtime/BlazeOptionHandler.java

```diff
@@ -578,6 +578,9 @@ private String getNotInRealWorkspaceError(Path doNotBuildFile) {
     return message;
   }
 
+  private static final Set<String> BUILD_COMMAND_ANCESTORS =
+      ImmutableSet.of("build", COMMON_PSEUDO_COMMAND, ALWAYS_PSEUDO_COMMAND);
```

Comment:
Can you also add a constant for `"build"` (and use it on line 617 also)?

Reply:
Done